apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue

```diff
@@ -4,18 +4,11 @@
     title="Go back to the homepage"
     to="/"
   >
-    <img
-      class="d-h42"
-      :alt="brandLogoAlt"
-      :src="brandLogo"
-    >
+    <svg-loader :illustration="true" name="dialpad-logo" />
```

Comment:
Is it possible to directly use `<dt-illustration name="dialpad-logo" />` rather than `svg-loder`?

Reply:
No, it doesn't work because the setup of the logo is done before the Vue components are registered